src/components/Navbar.tsx

```
@@ -0,0 +1,47 @@
+'use client'
+import React, { useState } from "react";
+import { HoveredLink, Menu, MenuItem, ProductItem } from "./ui/navbar-menu";
```

ESLint

[error] 'ProductItem' is defined but never used.

(@typescript-eslint/no-unused-vars)